spec/features/find/search/across_england/secondary_spec.rb

```diff
@@ -153,11 +153,11 @@ def when_i_select_the_secondary_subject
   end
 
   def then_i_should_see_the_visa_status_page
-    expect(page).to have_content('Do you have the right to work or study in the UK?')
+    expect(page).to have_content('Do you need visa sponsorship?')
   end
 
   def when_i_select_my_visa_status
-    choose 'Yes'
+    choose 'No'
```

Comment:
should we add a step `then_the_visa_filter_should_not_be_checked` or something related?

Reply:
Yep, also done here: https://github.com/DFE-Digital/publish-teacher-training/pull/3906/commits/c6753463b1483e05d89c47a335336de538a52a82